app/javascript/components/supplemental_application/actions.js

```diff
@@ -13,9 +13,18 @@ export const updateApplicationAction = async (application) => {
 }
 
 export const getAMIAction = async ({chartType, chartYear}) => {
-  const response = await apiService.getAMI({chartType, chartYear})
+  const response = await apiService.getAMI({ chartType, chartYear })
   if (response === false) {
     Alerts.error()
   }
   return response['ami']
 }
+
+export const addCommentsWithStatus = async (applicationId, comment, status) => {
+  const data = { status, comment, applicationId }
+  const response = await apiService.createLeaseUpStatus(data)
```

Comment:
@callaghanc I'm using `createLeaseUpStatus` here. The same method you used in the lease up table to change the status with the dropdown. Here is the code: https://github.com/Exygy/sf-dahlia-lap/blob/master/app/javascript/components/lease_ups/LeaseUpApplicationsTableContainer.js#L81 

It seems to be doing what we expect. Should be renamed `createLeaseUpStatus` to something more generic like `createFieldUpdateComment` ?

Reply:
Yes, let's rename it to `createFieldUpdateComment`, that more accurately describes what it does.